ldb/models.py

```diff
@@ -161,15 +161,15 @@ class Meta:
     )
 
     # Name
-    titles = models.CharField(_('titles'), max_length=20, blank=True)
+    titles = CharNullField(_('titles'), max_length=20, blank=True, null=True)
```

Comment:
Waarom deze wijziging naar nullable? Is `blank=True` niet voldoende?

Andere velden zijn alleen nullable als er een uniqueness constraint op staat (dan zal een blank als dubbele waarde niet mogen).

Reply:
Het valt me nu pas op dat je de review in het Nederlands deed en ik net in het engels reageerde op jouw Nederlands :')

Ben het hier wel eens met Mark. Dacht dat je hier zei dat het al `CharNullFields` waren

Reply:
Het waren geen `CharNullFields`, maar in mijn ogen is dit veel logischer (en sneller, de import nam veel minder tijd op). Dit omdat de CharField `` opslaat in de DB als er niks is ingevuld i.p.v. `<null>`.

Aan de andere kant is een empty string een indicatie dat het niet is ingevuld en een `<null>` value meer een indicatie dat het onbekend is.

Als we geen nullable values willen toestaan dan moeten we in de import functie hardcoden welke values van een `<null>` naar een `` geconverteerd moeten worden. Dit omdat uit een excel de value echt `<null>` is en niet ``. Het kan allebei.